packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3164,7 +3166,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             const stats: { getVersionDuration?: number; getSnapshotDuration?: number; } = {};
             const trace = Trace.start();
 
-            const versions = await this.storage.getVersions(versionId, 1);
+            const versions = await this.storage.getVersions(
+                versionId, 1, "refreshLatestSummaryAckFromServer", bypassCache);
```

Comment:
>refreshLatestSummaryAckFromServer [](http://example.com/codeflow?start=31&length=33)

Should we pass the scenario as an argument as well ?  Perhaps combined ? 
I wonder if there will be different scenarios where we can get the latest version

Reply:
Yes there are different scenarios like "Container_Load" etc. What do you mean  by combined argument?